pkg/operator/ceph/cluster/mon/spec_test.go

```diff
@@ -88,7 +88,7 @@ func testPodSpec(t *testing.T, dataDir string) {
 	assert.Nil(t, test_opceph.VerifyPodLabels("rook-ceph-mon", "ns", "mon", "a", pod.ObjectMeta.Labels))
 	assert.Equal(t, c.Namespace, pod.ObjectMeta.Labels["mon_cluster"])
 
-	assert.Equal(t, 3, len(pod.Spec.InitContainers))
+	assert.Equal(t, 2, len(pod.Spec.InitContainers))
```

Comment:
We had one init container just for the monmap?

Reply:
Yes.